### PR TITLE
Readme Update

### PR DESCRIPTION
**Pull request by Huddie**
Fixes some deadlinks
   - feature label was renamed to feature-request
   - setup guide link did not work on GitHawk

Small other changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ open Freetime.xcworkspace/
 
 This will install [apollo-codegen](https://github.com/apollographql/apollo-codegen) that is required to generate the GraphQL models.
 
-See the [setup guide](Setup.md) for details about API keys and environment variables.
+See the [setup guide](https://github.com/GitHawkApp/GitHawk/blob/master/Setup.md) for details about API keys and environment variables.
```
Comment (BasThomas) [inline README.md]: Why did you change this one?

Comment (Huddie) [reply README.md]: Didn’t work in GitHawk. Opened to GitHub 404 error

Comment (Sherlouk) [reply README.md]: Is there a bug raised for this?
